package.json

```diff
@@ -5,6 +5,8 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
+    "deploy:s3": "serverless client deploy --no-config-change --no-policy-change --no-cors-change",
```

Comment:
what the diff between this line and line #10, 'client:deploy' command?

Reply:
Hi! Seems there is no diff between them, they are do the same. I used this one, but I also ok with [rolling-scopes-school#10](https://github.com/rolling-scopes-school/nodejs-aws-fe/pull/10). Is it incorrect or anything else?

Reply:
When I've been doing the task I had little understanding what sls deploy does, so I wrote my own command.

Reply:
sls is just alias for serverless so its duplicated command

Reply:
Ok, I have removed duplicated scripts, ty for suggestions.